zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/controller/AuthenticationController.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.gateway.rest.controller;
+
+import io.camunda.authentication.entity.CamundaUserDTO;
+import io.camunda.authentication.service.CamundaUserService;
+import org.springframework.context.annotation.Profile;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Profile("auth-basic")
```

Comment:
How do web apps get a description of an authenticated user?

Reply:
From slack: 
> there will also be a OIDC user service

Reply:
We will eventually use this pattern also for OIDC (and LDAP) but it will be in a following PR :)